TLSharp.Core/TelegramClient.cs

```diff
@@ -258,11 +258,22 @@ public async Task<Boolean> SendTypingAsync(TLAbsInputPeer peer)
             return await SendRequestAsync<Boolean>(req);
         }
 
-        public async Task<TLAbsDialogs> GetUserDialogsAsync()
+        public async Task<TLAbsDialogs> GetUserDialogsAsync(int offset_date = 0, int offset_id = 0, TLAbsInputPeer offset_peer = null, int limit = 100)
```

Comment:
if it's a date, shouldn't the param type be of type Date, and let the GetUserDialogAsync convert the param to int under the hood?